utils/mailer.js

```diff
@@ -13,7 +14,7 @@ const transport = nodemailer.createTransport({
   },
 })
 
-const sendEmail = util.promisify(transport.sendMail.bind(transport))
+const sendEmail = options => transport.sendMail(options)
```

Comment:
If you don't pormisify it, it returns `undefined`, which will make `await` not wait for anything...

Reply:
Is that a scenario you encountered? It should return a promise if no callback is set and did when I tested it (it's why I left the line there, to make the one argument part more explicit).

Reply:
You are right, nodemailer docs miss the Promise part.